### DISCUSSION TITLE
Does anything special need to be done for Ubuntu 23.04?

### DISCUSSION THREAD

**Discussion by UntouchedWagons** [zbm-dev/zfsbootmenu#444] [Questions]
There's the instructions for [22.04](https://docs.zfsbootmenu.org/en/v2.2.x/guides/ubuntu/uefi.html), does anything special need to be done for 23.04?

**Comment by zdykstra** [comment]
The 22.04 directions likely translate almost directly to newer non-LTS releases, but I haven't attempted it. If you give it a shot, please let us know how it works for you!

**Comment by UntouchedWagons** [comment]
The guide for 22.04 worked just fine. Only changes were to replace any instances of `jammy` with `lunar` and remove the very last apt source
